src/hooks/useCart/useCart.test.tsx

```diff
@@ -38,8 +49,14 @@ const defaultProducts: Policy[] = [
       }
     },
     identifiers: [
-      { scannerType: "QR", label: "first" },
-      { scannerType: "QR", label: "last" }
+      {
+        ...defaultIdentifier,
+        label: "first"
```

Comment:
ehh, i'm a bit confused here, so there's 2 labels ah, one inside the defaultIdentifier, and one more outside? :O